app/src/main/java/swati4star/createpdf/fragment/ImageToPdfFragment.java

```diff
@@ -153,6 +161,8 @@ public void onAttach(Context context) {
     @Override
     public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
 
+        pref = mActivity.getApplicationContext ().getSharedPreferences("MyPref", MODE_PRIVATE);
```

Comment:
use PreferenceManager.getDefaultSharedPreferences(context) to get sharedpref instance. dont create new typr of prefs for this